keyboards/fortitude60/keymaps/default/keymap.c

```diff
@@ -0,0 +1,165 @@
+#include QMK_KEYBOARD_H
+#include "action_layer.h"
```

Comment:
Also, remove the additional includes here.  `QMK_KEYBOARD_H` includes this (well, it includes the keyboad's h file which includes quantum.h which includes these). 

So it's unnecessary, and cleaner looking.